endpoints-versions/src/main/scala/com/thenewmotion/ocpi/VersionsRoute.scala

```diff
@@ -70,9 +70,11 @@ class VersionsRoute(versions: => Map[VersionNumber, OcpiVersionConfig]) extends
 
   val VersionMatcher = Segment.flatMap(s => VersionNumber.opt(s))
 
-  def route(apiUser: GlobalPartyId)(implicit ec: ExecutionContext, mat: ActorMaterializer) = {
+
+  def route(apiUser: GlobalPartyId, securedConnection: Boolean = true)(implicit ec: ExecutionContext, mat: ActorMaterializer) = {
```

Comment:
Just wondering: how about ssl context? since you are now skipping traefik https checks?